ee/clickhouse/models/property.py

```diff
@@ -11,7 +11,7 @@
 from posthog.models.cohort import Cohort
 from posthog.models.property import Property
 from posthog.models.team import Team
-from posthog.utils import relative_date_parse
+from posthog.utils import is_valid_regex, relative_date_parse
 
 
 def parse_prop_clauses(
```

Comment:
Maybe worth adding a test for this handling in `ee/clickhouse/models/test/test_property.py`